lego/apps/events/views.py

```diff
@@ -167,12 +168,8 @@ def administrate(self, request, *args, **kwargs):
         event_data = populate_event_registration_users_with_grade(event_data)
         return Response(event_data)
 
-    @decorators.action(
-        detail=True, methods=["POST"], serializer_class=StripeTokenSerializer
-    )
+    @decorators.action(detail=True, methods=["POST"], serializer_class=BaseSerializer)
```

Comment:
Since the endpoint has breaking changes you need to deploy both frontend and backend at the same time and at a time when nobody uses the site. Otherwise a users payment might fail if you deploy while they are trying to pay.

You could also add a new endpoint and remove this one later to avoid breaking changes.

Reply:
We have to deploy this _before_ the next payment, so that should be no problem. (as no payments will go through now)